b/index.md

```diff
@@ -0,0 +1,6 @@
+## List of artists index b
```

Comment:
This should be a title heading, and only have 1 hash.
It also needs to match the format we used in the template.